force-app/tests/CI Recipes/DeprecatedAPIVersionScanner_Tests.cls

```diff
@@ -0,0 +1,94 @@
+@isTest
+private class DeprecatedAPIVersionScanner_Tests {
+    private static Integer apiFamilyColumnIndex;
+    private static Integer apiVersionColumnIndex;
+
+    private class ColumnNotFoundException extends Exception {
+    }
+
+    @IsTest
+    static void logsContainDeprecatedAPIVersions() {
+        Set<String> apiVersionsInUse = new Set<String>();
+        List<EventLogFile> logFiles = [
+            SELECT LogFile
+            FROM EventLogFile
+            WHERE EventType = 'ApiTotalUsage'
+        ];
+        if (logFiles.size() >= 1) {
+            // Check the log files for deprecated API versions
+            for (EventLogFile logFile : logFiles) {
+                apiVersionsInUse.addAll(
+                    buildSetOfReferencedAPIVersions(
+                        parseLogFile(logFile.LogFile.toString())
+                    )
+                );
+            }
+            System.assertEquals(
+                0,
+                apiVersionsInUse.size(),
+                'The following API versions are deprecated but still in use: ' +
+                String.join(new List<String>(apiVersionsInUse), ', ')
+            );
+        }
+        // implicit else here is that we have no logFiles to check.
```

Comment:
Don't you want to `System.assert(true)` so that the test does something in this case?